src/utils/redis.js

```diff
@@ -43,10 +43,11 @@ const redisTtlAsync = (...args) => {
 
 function getRedisIdentifier(token) {
 	const decodedToken = jwt.decode(token.replace('Bearer ', ''));
-	const { accountId, jti } = decodedToken; // jti - UID of the token
-
+	const { accountId, jti, exp } = decodedToken; // jti - UID of the token
+	const nowInSeconds = Math.floor(Date.now() / 1000);
+	const expirationInSeconds = exp - nowInSeconds;
 	const redisIdentifier = `jwt:${accountId}:${jti}`;
-	return redisIdentifier;
+	return { redisIdentifier, expirationInSeconds };
```

Comment:
I would recommend renaming this function. `getRedisIdentifier` is misleading if it returns more

Reply:
thanks for your recommendation, it's updated